lib/logstash/inputs/beats.rb

```diff
@@ -86,6 +86,9 @@ class LogStash::Inputs::Beats < LogStash::Inputs::Base
   # This option needs to be used with `ssl_certificate_authorities` and a defined list of CAs.
   config :ssl_verify_mode, :validate => ["none", "peer", "force_peer"], :default => "none"
 
+  # Time in milliseconds for an incomplete ssl handshake to timeout
+  config :ssl_handshake_timeout, :validate => :number, :default => 10000
```

Comment:
I am actually exposing a SSLHandler settings down to the consumer I though adding a test for it was unnecessary. I could be wrong WDYT?